extensions/wikidata/src/org/openrefine/wikidata/qa/scrutinizers/FormatScrutinizer.java

```diff
@@ -62,7 +81,12 @@ protected Pattern getPattern(PropertyIdValue pid) {
         if (_patterns.containsKey(pid)) {
             return _patterns.get(pid);
         } else {
-            String regex = _fetcher.getFormatRegex(pid);
+            String regex = null;
+            List<Statement> statementList = _fetcher.getConstraintsByType(pid, FORMAT_CONSTRAINT_QID);
+            if (!statementList.isEmpty()){
+                FormatConstraint constraint = new FormatConstraint(statementList.get(0));
+                regex = constraint.regularExpressionFormat;
+            }
```

Comment:
It would be good to take advantage of the new architecture to use all the format constraints declared on the property, not just the first one.